contrib/test/integration/build/runc.yml

```diff
@@ -4,7 +4,7 @@
   git:
     repo: "https://github.com/opencontainers/runc.git"
     dest: "{{ ansible_env.GOPATH }}/src/github.com/opencontainers/runc"
-    version: "c6e4a1ebeb1a72b529c6f1b6ee2b1ae5b868b14f"
+    version: "ab4a8191679806893207d9c1e507a77ff46dafaa"
```

Comment:
For sanity, do we want to bump [`vendor.conf`][1] and [the `Dockerfile`][2] to the same version?

I'm also not sure how much of runc we actually use, but we may want to synchronize our overlapping dependencies to avoid surprises when we call libcontainer.  For example, [we currently depend][3] on golang.org/x/sys golang/sys@9aade4d3a3b7e6d876cd3823ad20ec45fc035402, while [runc ab4a819167 depends on][4] golang/sys@7ddbeae9ae08c6a06a59597f0c9edbc5ff2444ce.  Those only differ by a few days, but tracking down dependency-divergence bugs sounds tedious enough that I'd rather just pull our sub-dependency values straight from our direct dependencies (and take the latest value in the event of diamond dependencies?).

[1]: https://github.com/kubernetes-incubator/cri-o/blob/214096b7ed4bc2f66f680161c97e18b9f5966111/vendor.conf#L24
[2]: https://github.com/kubernetes-incubator/cri-o/blob/214096b7ed4bc2f66f680161c97e18b9f5966111/Dockerfile#L60
[3]: https://github.com/kubernetes-incubator/cri-o/blob/214096b7ed4bc2f66f680161c97e18b9f5966111/vendor.conf#L67
[4]: https://github.com/opencontainers/runc/blob/ab4a8191679806893207d9c1e507a77ff46dafaa/vendor.conf#L21